### ISSUE TITLE
Specify Proxy Server

### ISSUE THREAD

**Issue by l0ckm4** [nickdnk/zerobounce-php#2]
How can you specify a proxy server to be used in the wrapper?  I can manually amend the zerobounce.php file to include the proxy but would like to leave that untouched and specify it in my own code.

Changes to zerobounce.php which works is:-

    public function __construct(string $apiKey, int $timeout = 15)
    {

        $this->apiKey = $apiKey;
        $this->client = new Client(
            [
                'base_uri'        => 'https://api.zerobounce.net/v2/',
                'timeout'         => $timeout,
                'connect_timeout' => 10,
		'proxy' => '192.168.0.200:3128'
            ]
        );
    }

**Comment by nickdnk** [comment]
We can just add an optional string parameter to the constructor if you wish.

Something like:

```php
public function __construct(string $apiKey, int $timeout = 15, ?string $proxy = null)
{

   $options = [
            'base_uri'        => 'https://api.zerobounce.net/v2/',
            'timeout'         => $timeout,
            'connect_timeout' => 10
        ];

    if ($proxy) {
      $options['proxy'] = $proxy;
    }

    $this->apiKey = $apiKey;
    $this->client = new Client($options);
}
```

**Comment by l0ckm4** [comment]
Sounds good to me - thanks

**Comment by nickdnk** [comment]
Released as 1.2.0.

Note that I opted for an array argument so you can pass anything supported by https://docs.guzzlephp.org/en/stable/request-options.html#proxy.
